pkgs/top-level/all-packages.nix

```diff
@@ -27591,9 +27591,11 @@ with pkgs;
 
   opentimestamps-client = python3Packages.callPackage ../tools/misc/opentimestamps-client {};
 
-  opentoonz = (qt514.overrideScope' (_: _: {
-    libtiff = callPackage ../applications/graphics/opentoonz/libtiff.nix { };
-  })).callPackage ../applications/graphics/opentoonz { };
+  opentoonz-libtiff = callPackage ../applications/graphics/opentoonz/libtiff.nix { };
+  opentoonz = qt5.callPackage ../applications/graphics/opentoonz {
```

Comment:
```suggestion
  opentoonz = let
    opentoonz-libtiff = callPackage ../applications/graphics/opentoonz/libtiff.nix { };
  in qt5.callPackage ../applications/graphics/opentoonz {
```

Reply:
I'd really suggest not exposing this overridden version in the main package set unless there's an explicit reason to do so.

Reply:
Ok.  Done.  Thanks!